lib/src/Pos/Web/Server.hs

```diff
@@ -273,29 +284,34 @@ nodeServantHandlers =
     -- :<|> getOurSecret
     -- :<|> getSscStage
 
-getLeaders :: Maybe EpochIndex -> WebMode ext SlotLeaders
+getLeaders :: MonadDBRead m => Maybe EpochIndex -> m SlotLeaders
```

Comment:
What's the point of all these `mtl` classes if you just run everything in `SomeMonstrousMonad` anyway?

Reply:
:upside_down_face: ...